besu/src/main/java/org/hyperledger/besu/cli/options/stable/PluginsConfigurationOptions.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.cli.options.stable;
+
+import static org.hyperledger.besu.cli.DefaultCommandValues.DEFAULT_PLUGINS_OPTION_NAME;
+
+import org.hyperledger.besu.cli.converter.PluginInfoConverter;
+import org.hyperledger.besu.cli.options.CLIOptions;
+import org.hyperledger.besu.cli.util.CommandLineUtils;
+import org.hyperledger.besu.ethereum.core.plugins.PluginConfiguration;
+import org.hyperledger.besu.ethereum.core.plugins.PluginInfo;
+
+import java.util.List;
+
+import picocli.CommandLine;
+
+/** The Plugins Options options. */
+public class PluginsConfigurationOptions implements CLIOptions<PluginConfiguration> {
+  @CommandLine.Option(
+      names = {DEFAULT_PLUGINS_OPTION_NAME},
+      description = "Comma-separated list of plugin names",
+      split = ",",
+      hidden = true,
+      converter = PluginInfoConverter.class,
+      arity = "0..*")
```

Comment:
Could make sense to have `arity=1..*`, to require at least one entry if specified?

Reply:
done